### PR TITLE
fix: init schema and form data on reset at DashboardWsidgetInputForm

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue [hunk 1]
```diff
@@ -223,6 +214,19 @@ watch([() => props.widgetConfigId, () => props.widgetKey, () => referenceStoreSt
...
+    // do nothing if still loading
+    if (loading) return;
+
+    // reset all states if required props are not given
+    if (!widgetConfigId) {
+        resetStates();
+        return;
+    }
+
+    initSchemaAndFormData(widgetConfigId, widgetKey);
```
Comment (yuda110) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue:229]: praise: 👍

**Review by yuda110 — APPROVED** [review]
LGTM!

**Review by sulmoJ — APPROVED** [review]
Clear~~

**Review by piggggggggy — APPROVED** [review]
LGTM! It's better!

**Pull request merged**